test/keyv.js

```diff
@@ -75,5 +75,29 @@ test.serial('.set(key, val, ttl) where ttl is "0" overwrites default tll option
 	tk.reset();
 });
 
+test.serial('Keyv uses json-buffer as default serializer', async t => {
```

Comment:
Sorry, I know I told you to write this but just realised this test isn't actually needed. We know it works because it's covered by the type tests in `keyvTestSuite`.